src/multicall.ts

```diff
@@ -9,29 +9,49 @@ export const multicallAddresses: { [ChainId: number]: string } = {
   [ChainId.POL]: '0x02817C1e3543c2d908a590F5dB6bc97f933dB4BD',
   [ChainId.BSC]: '0xa9193376D09C7f31283C54e56D013fCF370Cd9D9',
   [ChainId.DAI]: '0x67dA5f2FfaDDfF067AB9d5F025F8810634d84287',
-  [ChainId.FTM]: '0x22D4cF72C45F8198CfbF4B568dBdB5A85e8DC0B5',
-  [ChainId.ARB]: '0x80C7DD17B01855a6D2347444a0FCC36136a314de',
   [ChainId.OKT]: '0xF4d73326C13a4Fc5FD7A064217e12780e9Bd62c3',
+  [ChainId.FTM]: '0x22D4cF72C45F8198CfbF4B568dBdB5A85e8DC0B5',
   [ChainId.AVA]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
-  [ChainId.FSN]: '0x0769fd68dFb93167989C6f7254cd0D766Fb2841F',
-  [ChainId.KOV]: '0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696',
+  [ChainId.ARB]: '0x80C7DD17B01855a6D2347444a0FCC36136a314de',
+  [ChainId.HEC]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
+  // [ChainId.OPT]: NOT FOUND
   [ChainId.ONE]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
-  // [ChainId.OPT]: '',
+  [ChainId.FSN]: '0x0769fd68dFb93167989C6f7254cd0D766Fb2841F',
   [ChainId.MOR]: '0x270f2F35bED92B7A59eA5F08F6B3fd34c8D9D9b5',
-  // [ChainId.CELO]: '0x9aac9048fC8139667D6a2597B902865bfdc225d3',
-  [ChainId.HEC]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
+  // [ChainId.EXP]: '', // TODO
+  // [ChainId.TCH]: '', // TODO
+  // [ChainId.UBQ]: '', // TODO
+  // [ChainId.MET]: '', // TODO
+  // [ChainId.DIO]: '', // TODO
```

Comment:
Should this be part of this PR or do we want to resolve the TODOs later?

Reply:
we will add them later once they're available. so we remain them as they're.